include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -3162,6 +3162,26 @@ struct OrtApi {
   */
   ORT_API2_STATUS(SetGlobalCustomJoinThreadFn, _Inout_ OrtThreadingOptions* tp_options, _In_ OrtCustomJoinThreadFn ort_custom_join_thread_fn);
   /// @}
+
+  /** \brief Synchronize bound inputs. The call may be necessary for some providers, such as cuda,
+  *   in case the system that allocated bound memory operated on a different stream. However, the
+  *   operation is provider specific and could be a no-op.
+  *
+  * \param[inout] binding_ptr
```

Comment:
Does just 'inout' work? The Doxygen documentation says it should be 'in, out'

Reply:
It is a copy of the above